postgres/assets/configuration/spec.yaml

```
@@ -38,6 +38,11 @@ files:
       value:
         type: string
         display_default: postgres
+    - name: reported_hostname
+      description: |
+        Set the reported hostname for this instance.
```

The description should be more clear; try something like this:

```suggestion
        Set the reported hostname for this instance. This value overrides the hostname detected by the Agent and can be useful to set a custom hostname when connecting to a remote database through a proxy.
```

---

Updated